tests/gitmojify/test_gitmojify.py

```diff
@@ -211,6 +211,42 @@ def test_gitmojify_with_convert_prefixes():
     assert result.startswith("✨")
 
 
+SUMMARY = "branch 'main' into branch 'dev'"
+DESCRIPTION = "This is a description."
+
+
+@pytest.mark.parametrize(
+    ["message_in", "message_out", "should_raise"],
+    [
+        (f"Merge {SUMMARY}", f"{GJ_MERGE} merge: {SUMMARY}", False),
+        (f"Merge: {SUMMARY}", f"{GJ_MERGE} merge: {SUMMARY}", False),
+        (f"merge {SUMMARY}", ..., True),
+        (f"merge: {SUMMARY}", f"{GJ_MERGE} merge: {SUMMARY}", False),
+        (
+            f"Merge {SUMMARY}\n\n{DESCRIPTION}",
+            f"{GJ_MERGE} merge: {SUMMARY}\n\n{DESCRIPTION}",
+            False,
+        ),
+        ("Merge", ..., True),
+        ("Merge: ", ..., True),
+        ("merge: ", ..., True),
+        # TODO: expected is f"{GJ_MERGE} merge: \n\n{DESCRIPTION}" and should_raise is True
+        (f"Merge \n\n{DESCRIPTION}", f"{GJ_MERGE} merge: {DESCRIPTION}", False),
+    ],
+)
+def test_gitmojify_with_convert_prefixes_merge(
+    message_in: str, message_out: str, should_raise: bool
+):
+    """Test gitmojify with convert_prefixes and the Merge prefix."""
+    if not should_raise:
+        result = mojify.gitmojify(message_in, convert_prefixes=["Merge"])
+        assert result == message_out
+    else:
+        with pytest.raises(ValueError) as exc_info:
+            mojify.gitmojify(message_in, convert_prefixes=["Merge"])
+        assert str(exc_info.value) == "invalid commit message"
```

Comment:
```suggestion
        with pytest.raises(ValueError, match="invalid commit message"):
            mojify.gitmojify(message_in, convert_prefixes=["Merge"])
```

Reply:
Does this `match` actually compare whether the error messages are the same, or is it just a message for humans to read? 

I once tried deleting some text within the match to see if it would cause pytest to fail, but the test still passed.

Reply:
It uses `re.search`, so the message doesn't need to match completely, but needs to be contained in the error message.

Reply:
Got it. Since I only tried deleting text and retesting, no wonder the tests always passed. 😅